app/templates/components/owner/repositories.hbs

```diff
@@ -151,6 +151,7 @@
         </p>
       {{else}}
         <ExternalLinkTo
+          @target=''
```

Comment:
I'd like to suggest explicitly using `_self` for target:

```suggestion
          @target='_self'
```

What do you think?